packages/playground/blueprints/src/lib/steps/set-site-language.ts

```diff
@@ -19,6 +22,40 @@ export interface SetSiteLanguageStep {
 	language: string;
 }
 
+/**
+ * Returns the URL to download a WordPress translation package.
+ *
+ * If the WordPress version doesn't have a translation package,
+ * the latest "RC" version will be used instead.
+ */
+const getWordPressTranslationUrl = (wpVersion: string, language: string) => {
+	/**
+	 * The translation API provides translations for all WordPress releases
+	 * including patch releases.
+	 *
+	 * RC and beta versions don't have individual translation packages.
+	 * They all share the same "RC" translation package.
+	 *
+	 * Nightly versions don't have a "nightly" translation package.
+	 * So, the best we can do is download the RC translation package,
+	 * because it contains the latest available translations.
+	 *
+	 * The WordPress.org translation API uses "RC" instead of
+	 * "RC1", "RC2", "BETA1", "BETA2", etc.
+	 *
+	 * For example translations for WordPress 6.6-BETA1 or 6.6-RC1 are found under
+	 * https://downloads.wordpress.org/translation/core/6.6-RC/en_GB.zip
+	 */
+	const wpVersionString = versionStringToLoadedWordPressVersion(wpVersion);
```

Comment:
Note that if `wpVersion` is `"woah-this-is-cool"` then `wpVersionString` will also be `"woah-this-is-cool"`. What would be a sensible fallback here? Perhaps a dedicated version conversion logic would make more sense here?

Reply:
[8cf6394](https://github.com/WordPress/wordpress-playground/pull/1987/commits/8cf63941d5b24fb7d53f1d4c7f2861bb17b9ac00) will default translations to the latest WP version to ensure we can always have a valid URL.

Reply:
Since this already uses involved patterns, calling versionStringToLoadedWordPressVersion seem to add complexity. What would this look like without that call?

Reply:
It would look something like this [f1f5842](https://github.com/WordPress/wordpress-playground/pull/1987/commits/f1f58428c832bd94f60e4e0cb83eb9786d59a0a5)

I like this approach more, relying on `versionStringToLoadedWordPressVersion` could lead to issues if the function is changed, especially without tests.

Reply:
Lovely, looking good. Thank you!